docs/research/Orange_Pi_flasher.md

```diff
@@ -0,0 +1,79 @@
+# Orange Pi flasher
+
+## Contents
+
+<!-- TOC -->
+
+- [Orange Pi flasher](#orange-pi-flasher)
+    - [Contents](#contents)
+    - [Installing armbian on Orange Pi](#installing-armbian-on-orange-pi)
+    - [Installing flashrom on Orange Pi](#installing-flashrom-on-rpi)
+    - [Connection](#connection)
+    - [Flashing](#flashing)
+
+<!-- /TOC -->
+
+## Installing armbian on Orange Pi
+
+We need to prepare a SD card with armibian image on it first.
```

Comment:
This sentence is not coherent with the next one. I.e. "*we* need to prepare" and "extract it to *your* SD card". Are wa talking about "ourselves" or "them"? Be consistent.